rust/garbage_collector/src/operators/compute_versions_to_delete.rs

```diff
@@ -73,6 +73,7 @@ impl Operator<ComputeVersionsToDeleteInput, ComputeVersionsToDeleteOutput>
         let mut marked_versions = Vec::new();
         let mut oldest_version_to_keep = 0;
 
+        let mut version_file = version_file.as_ref().clone();
```

Comment:
this creates a new mutable copy of the version file
this operator is only used by v1 of GC